owncloudDomain/src/main/java/com/owncloud/android/domain/server/model/ServerInfo.kt

```diff
@@ -21,7 +21,7 @@ package com.owncloud.android.domain.server.model
 
 data class ServerInfo(
     val ownCloudVersion: String,
-    val baseUrl: String,
+    var baseUrl: String,
```

Comment:
> 😮 Im not sure about this.

> This is needed here:

> if (lastPermanentLocation != null) { serverInfo.baseUrl = WebdavUtils.trimWebdavSuffix(lastPermanentLocation) }

Yep, that's why I changed it from `val` to `var`


> What about doing something like
> serverInfo.copy(baseUrl = WebdavUtils.trimWebdavSuffix(lastPermanentLocation))

I think `copy` would make sense if we needed a second serverInfo object to do something with it, I do not see the advantages of using `copy` instead of changing the `serverInfo` property.